src/CribblyBackend/Controllers/StandingsController.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Threading.Tasks;
+using CribblyBackend.DataAccess.Models;
+using CribblyBackend.Network;
+using CribblyBackend.Services;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Primitives;
+using Serilog;
+
+namespace CribblyBackend.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class StandingsController : ControllerBase
+    {
+        private readonly IStandingsService StandingsService;
+        private readonly ILogger logger;
```

Comment:
🌵🌵 We've been a bit inconsistent with our naming schemes. Which one we go with is a matter of preference, but whatever we choose we should remain consistent. FWIW my preference is to have private fields be `_underscoreCamelCase` - the advantage with this is that the constructor doesn't need to use `this.` to differentiate between fields and parameters.

Either way though, let's pick one and stick with it 🙂